design/src/server/anatomy.md

```diff
@@ -365,14 +339,13 @@ Below we give an example of a ZST which can be provided to the builder, which al
 /// This _does_ implement [`Upgradable`] but produces a [`Service`] which always returns an internal failure message.
 pub struct FailOnMissingOperation;
 
-impl<P, Op, B> Upgradable<P, Op> for FailOnMissingOperation
+impl<Protocol, Operation, Exts, Body, Plugin> Upgradable<Protocol, Operation, Exts, Body, Plugin> for FailOnMissingOperation
```

Comment:
`Body` and `Plugin` have reappeared here too.

Reply:
Fixed in https://github.com/awslabs/smithy-rs/pull/1954/commits/b3de7a412bbb53dae4a52f9673acabec8867547b